apps/web/src/components/evaluations/Annotation/Form/IssuesSelector/index.tsx

```diff
@@ -160,19 +162,24 @@ export function IssuesSelector() {
           searchable
           removable
           width='auto'
+          align='center'
+          side='top'
+          sideOffset={8}
```

Comment:
Wrap the Selector here with the tooltip. Or add a `i` icon near the selector but do not overcomplicate Selector component

Reply:
Not solved in this PR but this totally wrong.